### PR TITLE
Changed version to 2.0.0-alpha

### PR DESCRIPTION
**Pull request by sfc-gh-astus**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added or updated automated unit tests to verify correctness of my new code.
   * [ ] I've added or updated integration tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are working by executing CLI's commands manually.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

File: src/snowcli/__about__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 from __future__ import annotations
 
-VERSION = "2.0.0-dev"
+VERSION = "2.0.0-alpha"
```
Comment (sfc-gh-pjob) [inline src/snowcli/__about__.py:3]: Maybe we should think about `2.0.0-alpha.1`? WDYT?
How do we want version releases with fixes?

Comment (sfc-gh-pjob) [reply src/snowcli/__about__.py]: @sfc-gh-turbaszek WDYT about this?

File: tests_e2e/conftest.py [hunk 2]
```diff
@@ -60,14 +61,14 @@ def _build_snowcli(venv_path: Path, test_root_path: Path) -> None:
 def _install_snowcli_with_external_plugin(
     venv_path: Path, test_root_path: Path
 ) -> None:
-    version = __about__.VERSION.replace("-", ".")
+    whl_file = _find_whl_file(os.listdir(test_root_path / f"../dist/"))
```
Comment (sfc-gh-pjob) [inline tests_e2e/conftest.py:64]: I can approve it as temporary solution but I think we should consider hardcoding an expected file name here.

Comment (sfc-gh-astus) [reply tests_e2e/conftest.py:64]: I will search for some solution after release

Comment (sfc-gh-pjob) [reply tests_e2e/conftest.py:64]: Ok, I forgot that we build snowcli in a venv, so I'm not sure we need any better solution :)

**Pull request merged**